homeassistant/components/climate/wink.py

```diff
@@ -453,12 +455,131 @@ def fan_list(self):
         """Return a list of available fan modes."""
         return [SPEED_LOW, SPEED_MEDIUM, SPEED_HIGH]
 
-    def set_fan_mode(self, mode):
+    def set_fan_mode(self, fan):
         """Set fan speed."""
-        if mode == SPEED_LOW:
+        if fan == SPEED_LOW:
             speed = 0.4
-        elif mode == SPEED_MEDIUM:
+        elif fan == SPEED_MEDIUM:
             speed = 0.8
-        elif mode == SPEED_HIGH:
+        elif fan == SPEED_HIGH:
             speed = 1.0
         self.wink.set_ac_fan_speed(speed)
+
+
+class WinkWaterHeater(WinkDevice, ClimateDevice):
+    """Representation of a Wink water heater."""
+
+    def __init__(self, wink, hass, temp_unit):
+        """Initialize the Wink device."""
+        super().__init__(wink, hass)
+        self._config_temp_unit = temp_unit
```

Comment:
This doesn't seem to be used.

Reply:
Looks like you are correct, left over from a copy/paste.